src/pages/components/code-snippet/accessibility.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Code snippet
+description: Code snippets are small blocks of reusable code that can be inserted in a code file.
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListInput,
+  StructuredListCell,
+  OrderedList,
+  ListItem,
+} from 'carbon-components-react';
+
+<AnchorLinks>
+  <AnchorLink>How it works</AnchorLink>
+  <AnchorLink>Accessibility considerations</AnchorLink>
+  <AnchorLink>Resources</AnchorLink>
+  <AnchorLink>Accessibility testing</AnchorLink>
+</AnchorLinks>
+
+<PageDescription>
```

Comment:
Can you move the `<PageDescription>` above the anchorlinks? Looks fine to me besides that.